Examples/VaporExample/Package.swift

```diff
@@ -1,15 +1,15 @@
-// swift-tools-version:5.2
+// swift-tools-version:5.5
 import PackageDescription
 
 let package = Package(
     name: "VaporExample",
     platforms: [
-        .macOS(.v10_15)
+        .macOS(.v12)
     ],
     dependencies: [
-        .package(url: "https://github.com/vapor/vapor", .upToNextMajor(from: "4.7.0")),
+        .package(url: "https://github.com/vapor/vapor", .upToNextMajor(from: "4.50.0")),
         .package(url: "https://github.com/vapor/leaf", .upToNextMajor(from: "4.0.0")),
-        .package(url: "https://github.com/mongodb/mongodb-vapor", .upToNextMajor(from: "1.0.0"))
+        .package(url: "https://github.com/mongodb/mongodb-vapor", .branch("async-await"))
```

Comment:
all this branch does is depend on the driver main branch so we get the new APIs. there are no code changes needed